lib/ui/widgets/haptics_container.dart

```diff
@@ -0,0 +1,25 @@
+// FLUTTER / DART / THIRD-PARTIES
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+class HapticsContainer extends StatefulWidget {
```

Comment:
Could you add a little description of the widget

Reply:
I added one, is it ok this way?